mne/viz/raw.py

```diff
@@ -728,10 +728,10 @@ def plot_raw_psd(raw, tmin=0., tmax=np.inf, fmin=0, fmax=np.inf, proj=False,
 
     for ii, (picks, title, ax) in enumerate(zip(picks_list, titles_list,
                                                 ax_list)):
-        psds, freqs = compute_raw_psd(raw, tmin=tmin, tmax=tmax, picks=picks,
-                                      fmin=fmin, fmax=fmax, proj=proj,
-                                      n_fft=n_fft, n_overlap=n_overlap,
-                                      n_jobs=n_jobs, verbose=None)
+        # XXX name conflict: plot_raw_psd, and psd_raw...
+        psds, freqs = psd_raw(
```

Comment:
the former names `compute_raw_psd` matched these plotting functions `plot_raw_psd`. I think doing `psd_raw` runs a bit counter to the way we have named some of these other functions...